python-sdk/langwatch/src/langwatch/__init__.py

```
@@ -0,0 +1,79 @@
+        raise err
+except ImportError:
+    pass
+
+dspy = None
+if dspy_available:
+    try:
+        from langwatch.dspy import langwatch_dspy
+
+        dspy = langwatch_dspy
+    except ImportError:
+        warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)